packages/cactus-common/src/main/typescript/signer-key-pairs.ts

```diff
@@ -0,0 +1,26 @@
+import crypto from "crypto";
+import secp256k1 from "secp256k1";
+
+export interface ISignerKeyPairs {
```

Comment:
Would rename this to be singular because an instance of this stores a single key pair (which does consist of two keys, but the pair itself is just the one)

Reply:
Rename the interface.